pkg/controller/submariner/submariner_controller.go

```diff
@@ -277,6 +277,8 @@ func newEnginePodTemplate(cr *submarinerv1alpha1.Submariner) corev1.PodTemplateS
 						{Name: "SUBMARINER_DEBUG", Value: strconv.FormatBool(cr.Spec.Debug)},
 						{Name: "SUBMARINER_NATENABLED", Value: strconv.FormatBool(cr.Spec.NatEnabled)},
 						{Name: "SUBMARINER_BROKER", Value: cr.Spec.Broker},
+						{Name: "SUBMARINER_CABLE", Value: cr.Spec.CableType},
+						{Name: "SUBMARINER_DRIVER", Value: cr.Spec.DriverImpl},
```

Comment:
Again we could leave it back to convention over a single env var to reduce complexity/stuff moving around.